### PR TITLE
doc: Introduce release note for 1.22.0 release

### PR DESCRIPTION
**Pull request by slandelle**

### CODE REVIEW

File: data/variables.toml
Comment (Isammoc) [inline data/variables.toml:1]: commit message:
`release not` => `release notes`

File: bin/entrypoint.sh [hunk 1]
```diff
@@ -131,7 +131,7 @@ prepare () {
     npm install
   
     #               # repository  # branch  # remote            # local
-    hugo_structure  "gatling"     "3.11"    "src/docs/content"  "content"
+    hugo_structure  "gatling"     "main"    "src/docs/content"  "content"
```
Comment (Isammoc) [inline bin/entrypoint.sh:134]: back to main?

Should we have a `stable` branch (default one)
and a branch by version?

**Review by Isammoc — COMMENTED** [review]
co

File: bin/entrypoint.sh (hunk 1 above)
Comment (slandelle) [reply bin/entrypoint.sh:134]: We only create the maintenance branch once we introduce a new feature.
We already have enough commits to port around.

**Pull request merged**